Formula/tiny-fugue.rb

```diff
@@ -14,13 +14,16 @@ class TinyFugue < Formula
   conflicts_with "tee-clc", :because => "both install a `tf` binary"
 
   depends_on "libnet"
+  depends_on "openssl"
   depends_on "pcre"
 
   # pcre deprecated pcre_info. Switch to HB pcre-8.31 and pcre_fullinfo.
   # Not reported upstream; project is in stasis since 2007.
   patch :DATA
 
   def install
+    ENV.append "CPPFLAGS", "-I#{Formula["openssl"].opt_prefix}include"
```

Comment:
It looks only in /usr/local by default (it seems), and openssl is one of those packages that do not stick their include/libs there. Does depends_on implicitly add these? (I added the 3 lines and it worked, without, it didn't, even with openssl packjage installed.)